.gitlab/e2e/e2e.yml

```
@@ -124,21 +127,26 @@ k8s-e2e-otlp-main:
       annotations:
         - $EXTERNAL_LINKS_PATH
 
+.needs_new_e2e_template:
```

```suggestion
.new_e2e_go_deps:
```
As it's on a `needs` clause there is maybe no need to repeat `needs`, and I prefer to mention explicitly the role of this clause wdyt?

---

sgtm